charts/shoot-core/components/charts/metrics-server/templates/deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: metrics-server
   namespace: kube-system
   labels:
+    networking.gardener.cloud/from-seed: allowed
```

Comment:
Metric server would also need DNS access as well.
Also access to the Nodes CIDR

Reply:
thanks, i've updated the file